packages/now-go/util/analyze.go

```diff
@@ -195,11 +194,36 @@ func main() {
 			continue
 		}
 		if fn.Name.IsExported() == true {
-			// find a valid `net/http` handler function
+			// find a valid `http.HandlerFunc` handler function
+			params := rf[fn.Type.Params.Pos()-offset : fn.Type.Params.End()-offset]
+			validHandlerFunc := (strings.Contains(string(params), "http.ResponseWriter") &&
+				strings.Contains(string(params), "*http.Request"))
```

Comment:
Previously this was checking for `*http.Request http.ResponseWriter`. Why was it changed? Do you want to be able to match 3 parameters?

Reply:
It was bug that I found, it wasn't check as we intended it to be.